en/docs/learn/configuring-saml2-web-single-sign-on.md

```diff
@@ -237,7 +237,7 @@ To configure SAML2 Web SSO:
    <p class="admonition-title">Warning</p>
     <p>To configure IdP EntityID Alias described below, apply the <strong>5102 WUM update</strong> to WSO2 IS 5.8.0 using the WSO2 Update Manager (WUM). To deploy a WUM update into production, you need to have a paid subscription. If you do not have a paid subscription, you can use this feature with the next version of WSO2 Identity Server when it is released. For more information on updating WSO2 Identity Server using WUM, see <a href="https://docs.wso2.com/display/updates/Getting+Started">Getting Started with WUM</a> in the WSO2 Administration Guide.</p></div>
     <p>This value can override the value of Identity Provider Entity ID specified under SAML SSO Inbound Authentication configuration in Resident IdP. The Identity Provider Entity ID is used as the issuer of the SAML responses generated from IS. By default, all the SAML responses issued by IS will have the issuer value similar to the Identity Provider Entity ID in Resident IdP’s SAML SSO inbound authentication configuration. But if you want that value to be unique for your SAML SP configuration, you can specify the value here, so that the IdP Entity ID will be overridden with this IdP Entity ID Alias value.</p>
-    <p>In Travelocity sample SP, this value can be set by modifying SAML2.IdPEntityId value mentioned in the <a href="http://travelocity.com/WEB-INF/classes/travelocity.properties">travelocity.com/WEB-INF/classes/travelocity.properties</a> file, so that it reflects the value of the IdP Entity ID Alias you define in the SAML SP configuration.</p>
+    <p>In Travelocity sample SP, this value can be set by modifying SAML2.IdPEntityId value mentioned in the `<travelocity.com-hpme>/WEB-INF/classes/travelocity.properties` file, so that it reflects the value of the IdP Entity ID Alias you define in the SAML SP configuration.</p>
```

Comment:
Is this supposed to be <travelocity.com-home>?

Reply:
fixed